index.js

```diff
@@ -1,7 +1,149 @@
-const getRandomNumber = (min = 0, max = 100) => {
-  const randomNumber = Math.floor(Math.random() * (max - min) + min);
+const http = require('http');
+const { db } = require('./database/database');
+const querystring = require('querystring');
+const { userNotFound, getHtmlBlock } = require('./utils/utils');
 
-  console.log(randomNumber);
-}
+const server = http.createServer((request, response) => {
+  const [url, query] = request.url.split('?');
+  const params = querystring.parse(query);
+  const { id, name, email, hobbies } = params;
```

Comment:
Not very RESTful: I'd recommend your server to accept the following urls:
`/users`
`/users/:id`
`/users/:id/hobbies`
usually, query params are used to specify request query (for example, filtering)

Reply:
Additionally, your server should accept different request methods (i.e POST, PUT or PATCH, DELETE)